src/Header.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { Form } from 'react-bootstrap';
 
 class Header extends React.Component{
     render(){
         return(
             <header>
                 <h1>Horned Animals Galore</h1>
+                <Form>
+                    <Form.Group>
+                        <Form.Label>Number of Horns</Form.Label>
+                        <Form.Select onChange={this.props.handleFormSubmit}>
+                            <option value="1">1</option>
+                            <option value="2">2</option>
+                            <option value="3">3</option>
+                            <option value="100">4+</option>
+                        </Form.Select>
+                    </Form.Group>
+                </Form>
             </header>
         );
     }
```

Comment:
Good work on this!

- What I am thinking here is that perhaps we could leave Header component as it's own component and decouple the Form from it a bit by creating a separate component to hold the Form.
- The reason I am considering this approach is primarily for easier reusability (but I could be going into overkill mode here since this is a small app...yes that is most likely the case, just some food for thought :))
-  Ideally we could also try to make this a controlled input, basically the  form element's value would be controlled by React. The form value would be determined by the state value.